tests/python/gaia-ui-tests/gaiatest/apps/music/regions/sublist_view.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+from gaiatest.apps.music.regions.player_view import PlayerView
+
+
+class SublistView(Base):
+
+    _album_sublist_view_locator = (By.CSS_SELECTOR, 'iframe.active[src*="/views/album-detail/index.html"]')
+    _artist_sublist_view_locator = (By.CSS_SELECTOR, 'iframe.active[src*="/views/artist-detail/index.html"]')
+    _album_list_locator = (By.ID, 'list')
+    _first_song_locator = (By.CLASS_NAME, 'gfl-item first')
+
+    def __init__(self, marionette, sublist_type):
+        Base.__init__(self, marionette)
+        if sublist_type == 'album':
```

Comment:
for consistency, if we break down list view, we should break down sublist view as well (the src are distinct too)